packages/lpa-questionnaire-web-app/src/views/includes/head.njk

```diff
@@ -1,17 +1,19 @@
-{% block head %}
-
-  <!--[if lte IE 8]><link href="/public/stylesheets/main-ie8.css" rel="stylesheet" type="text/css" /><![endif]-->
-  <!--[if gt IE 8]><!--><link href="/public/stylesheets/main.css" media="all" rel="stylesheet" type="text/css" /><!--<![endif]-->
-
-
-  <!-- Global site tag (gtag.js) - Google Analytics -->
-  <script data-cy="Google Analytics" async src="https://www.googletagmanager.com/gtag/js?id={{ googleAnalyticsId | safe }}"></script>
-  <script>
-    window.dataLayer = window.dataLayer || [];
-    function gtag(){dataLayer.push(arguments);}
-    gtag('js', new Date());
-
-    gtag('config', '{{ googleAnalyticsId | safe }}');
-  </script>
-
-{% endblock head %}
+{% block head %}
+
+  <!--[if lte IE 8]><link href="/public/stylesheets/main-ie8.css" rel="stylesheet" type="text/css" /><![endif]-->
+  <!--[if gt IE 8]><!--><link href="/public/stylesheets/main.css" media="all" rel="stylesheet" type="text/css" /><!--<![endif]-->
+
+
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script data-cy="Google Analytics" async src="https://www.googletagmanager.com/gtag/js?id={{ googleAnalyticsId | safe }}"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+
+    gtag('config', '{{ googleAnalyticsId | safe }}');
+  </script>
+
+  <meta name="appealReplyId" content="{{ appealReply.id }}">
```

Comment:
Meta should really be declared at the top of the head (know not possible, but should at least be above our generated content. Also should it have an if around it?